feed-integration-webapp/pom.xml

```diff
@@ -0,0 +1,357 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why use maven? We could very well use gradle which is way more performant and overall much better